pkg/operator/rbaccontroller/rbaccontroller.go

```diff
@@ -0,0 +1,100 @@
+package rbaccontroller
+
+import (
+	"context"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	listerrbacv1 "k8s.io/client-go/listers/rbac/v1"
+	"k8s.io/klog"
+)
+
+const (
+	// annotation used to indicate if unauthenticated user needs to be
+	// removed from the list of subjects for a cluster role binding
```

Comment:
hmm, this is still very specific to the bug, not generic. Code comments should be generic, i.e. describe the mechanism, even if the mechanism is used to fix the bug.

Reply:
gave another shot at it. please take a look.